daemon/container_unix.go

```diff
@@ -242,6 +242,12 @@ func populateCommand(c *Container, env []string) error {
 		}
 	}
 
+	var wDevices []string
+	for _, weightDevice := range c.hostConfig.BlkioWeightDevice {
```

Comment:
can we have this as a WeightDevice struct method like String()? just wondering, don't know what's other think about this